README.md

```diff
@@ -1,69 +1,64 @@
-# Voila Editor
+# voila-editor
 
-A JupyterLab extension to create voila dashboards.
+![Github Actions Status](https://github.com/.../workflows/Build/badge.svg)[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/.../master?urlpath=lab)
 
-[![Binder](https://mybinder.org/badge_logo.svg)]()
+A JupyterLab extension to create voila dashboards.
 
 ## Requirements
 
-* python >= 3.7
-* JupyterLab >= 3.0
-* npm >= 6.13.4
+- JupyterLab >= 3.0
 
 ## Install
 
 ```bash
-# Create a new environment with the dependencies
-mamba create -n test -c conda-forge python nodejs jupyterlab=3.0.0b0
-conda activate test
+pip install voila-editor
 ```
 
 ## Contributing
 
-### Install
+### Development install
+
+Note: You will need NodeJS to build the extension package.
 
 The `jlpm` command is JupyterLab's pinned version of
 [yarn](https://yarnpkg.com/) that is installed with JupyterLab. You may use
 `yarn` or `npm` in lieu of `jlpm` below.
 
 ```bash
 # Clone the repo to your local environment
-git clone https://github.com/...
-# Move to jupyterlab-ros directory
-cd jupyterlab-ros
+# Change directory to the voila-editor directory
+
+# create a new environment
+conda create -n voila-editor -c conda-forge python nodejs -y
```

Comment:
We can keep `mamba` here too

Reply:
Yep, we should use mamba.
Thanks!! I'll continue working on editor this afternoon.

Reply:
Nice. We can update the dev instructions in another PR (tracked in https://github.com/hbcarlos/voila-editor/issues/4)